examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -554,6 +554,9 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
             if ($scope.polygons.statePolygons) {
               $scope.map.removeLayer($scope.polygons.statePolygons);
             }
+            if ($scope.polygons.statePolygons) {
+              $scope.map.removeLayer($scope.polygons.statePolygons);
+            }
```

Comment:
Why is this needed? Seems a duplicate code?

Reply:
Sorry I committed from a wrong folder.